devbuild.sh

```diff
@@ -187,11 +188,48 @@ while :; do
   shift
 done
 
+# build conda and conda environments, if requested.
+set -x
+if [ "${BUILD_CONDA}" = "on" ] ; then
+  if [ ! -d "${CONDA_BUILD_DIR}" ] ; then
+    os=$(uname)
+    test $os == Darwin && os=MacOSX
+    hardware=$(uname -m)
+    installer=Miniforge3-${os}-${hardware}.sh
+    curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/${installer}"
+    bash ./${installer} -bfp "${CONDA_BUILD_DIR}"
+    rm ${installer}
+  fi
+
+  source ${CONDA_BUILD_DIR}/etc/profile.d/conda.sh
+  # Put some additional packages in the base environment on MacOS systems
+  if [ "${os}" == "MacOSX" ] ; then
+    mamba install -y bash coreutils sed
```

Comment:
@christinaholtNOAA I see you're installing coreutils but not gsed. Previously @natalie-perlin was installing gsed, but I think there was some question is to whether this was necessary. Just making sure its omission isn't an oversight.

Reply:
I believe that gsed was required for using brew, as it was documented in the UG, but when installing with conda, it's sed. 

In this case, I think the conda version of coreutils (instead of brew) also gets us `readlink` instead of `greadlink`. We might be able to simplify some things in the workflow scripts with this new environment.